src/functional/resources/repo-config.csv

```diff
@@ -1,4 +1,4 @@
-Repository's Location,Branch,Ignore Glob List
-https://github.com/reposense/testrepo-Beta.git,master,docs**
-https://github.com/reposense/testrepo-Charlie.git,master,
-https://github.com/reposense/testrepo-Delta.git,master,
+Repository's Location,Branch,Ignore Glob List,Ignore standalone config.json
```

Comment:
Since this is like an enabling/disabling option, perhaps `Disable standalone config` would be more appropriate.
Another reason is that it avoids confusion with `Ignore Glob List`